vNext/test/Tester/project.json

```diff
@@ -6,7 +6,31 @@
     "Newtonsoft.Json": "9.0.1",
     "xunit": "2.1.0",
     "xunit.runner.visualstudio": "2.1.0",
-    "Xunit.SkippableFact": "1.2.14"
+    "Xunit.SkippableFact": "1.2.14",
+
+
+
+
+    "System.Collections.Immutable": "1.2.0",
```

Comment:
Added a space here to separate the test dependencies (cannot add comments in a json file).
These dependencies are currently only needed because of our csproj hack, but should be gone after we remove the hack, and become normal transitive dependencies